x-pack/plugins/security_solution/public/common/components/hover_actions/actions/show_top_n.tsx

```diff
@@ -23,8 +29,8 @@ const SHOW_TOP = (fieldName: string) =>
   });
 
 interface Props {
-  /** `Component` is only used with `EuiDataGrid`; the grid keeps a reference to `Component` for show / hide functionality */
```

Comment:
Why remove the comment?

Reply:
Because it is not only used in EuiDataGrid. We also pass EuiContextMenuItem to `Component` prop when it is rendered in takeActionDropdown in flyout. I'll put back the comment and update it.